src/Symfony/Component/VarDumper/Caster/GmpCaster.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\VarDumper\Caster;
+
+/**
+ * Transform a GMP object to a integer or a string. It need the gmp php extension.
+ *
+ * @author Amrouche Hamza <hamza.simperfit@gmail.com>
+ */
+class GmpCaster
+{
+    public static function castInt(\GMP $value): array
```

Comment:
see other casters: that not their typical signature, we should use the same here also